02week/bio/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>Nick Andrew Gonzalez Biography</title>
+  <link href="css/style.css" rel="stylesheet" type="text/css" />
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```

Comment:
Nice, I like how you were able to include an external library of icons.
I would recommend that all those libraries and such should go first and at last your css file. To prevent overlapping and conflicts